examples/spark-s3-to-postgres/great_expectations/config/config_variables.yml

```diff
@@ -0,0 +1,16 @@
+# This config file supports variable substitution which enables: 1) keeping
```

Comment:
Is it of any use atm? Or for future functionality

Reply:
It is not used at the moment but referenced in the great_expectations.yml so I thought it would be good to have it.